src/DotNetty.Codecs.Redis/Messages/RedisCodecUtil.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace DotNetty.Codecs.Redis.Messages
+{
+    using System;
+    using System.Diagnostics.Contracts;
+    using System.Globalization;
+    using System.Text;
+    using DotNetty.Buffers;
+
+    static class RedisCodecUtil
+    {
+        internal const char RedisSimpleString = '+';
+        internal const char RedisError        = '-';
+        internal const char RedisInteger      = ':';
+        internal const char RedisBulkString   = '$';
+        internal const char RedisArray        = '*';
+
+        internal static byte[] GetBytes(this IRedisMessagePool messagePool, long value)
+        {
+            Contract.Requires(messagePool != null);
+
+            byte[] bytes;
+            if (!messagePool.TryGetBytes(value, out bytes))
+            {
+                bytes = LongToAsciiBytes(value);
+            }
+
+            return bytes;
+        }
+
+        internal static SimpleStringRedisMessage GetSimpleStringMessage(this IRedisMessagePool messagePool, IByteBuffer byteBuffer)
```

Comment:
I have moved some parts inline with encoder/decoder. The reason this is here was because I was marking around with ByteOrder options there.